### ISSUE TITLE
`role-supports-aria-props` documentation page displays a non working sample of code

### ISSUE THREAD

**Issue by n-chardon** [jsx-eslint/eslint-plugin-jsx-a11y#938]
The following code is displayed as the "Good" example of `role-supports-aria-props` rule : 

```
<ul role="radiogroup" aria-required aria-labelledby="foo">
    <li tabIndex="-1" role="radio" aria-checked="false">Rainbow Trout</li>
    <li tabIndex="-1" role="radio" aria-checked="false">Brook Trout</li>
    <li tabIndex="0" role="radio" aria-checked="true">Lake Trout</li>
</ul>
```

However, this code does not work as intended, because the `aria-required` attribute has a "false" default value: it should be written as `aria-required="true"` to work properly with assistive technology.

I would recommend that the documentation be updated with a working example.

**Comment by ljharb** [comment]
In HTML, does `aria-required="aria-required"` not work properly?
